modules/plugin/imagepyramid/src/main/java/org/geotools/gce/imagepyramid/ImagePyramidFormat.java

```diff
@@ -206,20 +206,15 @@ public boolean accepts(Object source, Hints hints) {
             }
```

Comment:
I though we were just passing everything down as-is? (besides the grid geometry and overview policy, which are used locally to figure out which mosaic reader to use). Can you make an example of a parameter that we need to "manage" or "parse"?